src/pyTRLCConverter/base_converter.py

```diff
@@ -0,0 +1,140 @@
+""" Converter base class which does the argparser handling and provides helper functions.
+
+    Author: Norbert Schulz (norbert.schulz@newtec.de)
+"""
+
+# pyTRLCConverter - A tool to convert PlantUML diagrams to image files.
+# Copyright (c) 2024 - 2025 NewTec GmbH
+#
+# This file is part of pyTRLCConverter program.
+#
+# The pyTRLCConverter program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software Foundation,
+# either version 3 of the License, or (at your option) any later version.
+#
+# The pyTRLCConverter program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with pyTRLCConverter.
+# If not, see <https://www.gnu.org/licenses/>.
+
+# Imports **********************************************************************
+import os
+from typing import Optional
+from pyTRLCConverter.abstract_converter import AbstractConverter
+from pyTRLCConverter.ret import Ret
+from pyTRLCConverter.trlc_helper import Record_Object
+
+class BaseConverter(AbstractConverter):
```

Comment:
Divider between imports and classes is missing, as well as others too. Please consider to add them according to the module template.